.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,38 @@
+name: E4E Packaging Workflow
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+        # order operating systems from best to worst
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get E4E package version
+        id: npm-version
+        uses: nyaa8/package-version@v1.0.3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install Dependencies
+        run: npm ci
+        env:
+          NODE_OPTIONS: --max_old_space_size=4096
+
+      - name: Package VSIX
+        run: npx vsce package
+      - run: mv ${{ env.PROJECT_NAME }}-${{ env.PACKAGE_VERSION }}.vsix ${{ env.PROJECT_NAME }}_latest.vsix
```

Comment:
Is it possible to save the packaged VSIX file somehow in the GH actions?

For example, in Jenkins, I can archive the files within the jobs, so they can be achieved (downloaded) from UI.

Reply:
Is it possible to use the packaged VSIX within Eclipse?

I believe, there is some rule, that according to Eclipse rules, the VSIX file should be packaged in some US servers only, right? @VitGottwald can I ask you to clarify this moment, if possible?

Reply:
I added a bit to upload the artifact to GH, but since space in GH is limited, I think eventually we should just upload it to Artifactory or wherever.

@VitGottwald we still need your input on point 2 though...

Reply:
Hmm, does the GH cleans the outdated artefacts by itself?

I would leave it also in the GH actions, because if there will be troubles with uploading it into marketplace or openVsx - we should still have an artefact to upload it manually.

Reply:
I will look into it...I think it has some time limit, but I will have to read the doc again

Reply:
We can configure the retention time of artifacts [here](https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/configuring-the-retention-period-for-github-actions-artifacts-and-logs-in-your-repository).